app/model/exchange/EventAllocations.scala

```diff
@@ -27,9 +28,12 @@ object EventAssessorAllocationsSummaryPerSkill {
   implicit val format = Json.format[EventAssessorAllocationsSummaryPerSkill]
 }
 
-case class EventWithAllocationsSummary(event: Event,
-                                       confirmedCandidates: Int,
-                                       assessorAllocations: List[EventAssessorAllocationsSummaryPerSkill])
+case class EventWithAllocationsSummary(
+  date: LocalDate,
```

Comment:
It should be refactored to a flatter structure.  I need it here to generically handle it in the calendar builder.